11ty/definitions/adhd.md

```diff
@@ -0,0 +1,68 @@
+
+### Executive Dysfunction
+Per [ADDitude](https://www.additudemag.com/what-is-executive-function-disorder/), Executive Dysfunction affects a person's ability to "engage in goal-directed action." This typically manifests in difficulty planning, completing tasks, and staying organized. This may mean someone with ADHD frequently has trouble starting chores, for example. In other cases, people with ADHD find it difficult to break large goals into small, actionable tasks.
+
+### Working Memory Issues
+People with ADHD often have issues with their working memory. Working memory is important for decision-making and reasoning - it allows people to keep and reference multiple thoughts quickly when synthesizing information and also when context-switching. A deficit in working memory means that people with ADHD can seem forgetful and it makes context-switching especially difficult. It can also result in someone bouncing around between several tasks, especially when those tasks are mundane or uninteresting.
+
+### Emotional Dysregulation
+ADHD makes it hard to regulate emotional reactions. According to [How to ADHD](https://www.youtube.com/watch?v=2kew2JhKq3Y), Emotional Dysregulation is often overlooked, but it can be partly attributed to deficits in inhibition, self-soothing, focus, and working memory. In children, this can cause tantrums. Adults with ADHD are often expressive, excitable, and they often show atypically intense emotion.
+
+### Distractibility
+Distractibility is one of the "hallmark" traits people think of when they think of ADHD. Because ADHD causes interest-driven behavior, people with ADHD often have trouble maintaining focus on uninteresting tasks. This means people with ADHD often jump between several tasks, and it can be very easy for them to become distracted by other things. Common distractors are loud sounds, other people, the Internet, and bright lights. Everyone experiences distraction at some point. However, for people with ADHD, distraction can be nearly constant and it disrupts focused, deep work.
+
+Distractibility also has social consequences - in conversation, it can seem to others that someone with ADHD is not listening or does not care. A common behavior to counteract distractibility is ["stimming"](https://www.healthline.com/health/autism/stimming), which is commonly associated with Autism. Many people with ADHD use fidget devices or other tools to stimulate their senses and keep themselves focused on a task or a conversation. Stimming is a natural reaction to anxiety or sensory overload, and it can help regulate focus.
```

Comment:
@bnb @tatianamac I added a few extra sections here to cover some of the common behaviors and traits of ADHD. I'm having a bit of difficulty gathering my thoughts into words right now (thanks, ADHD!), so I don't know how accurate my descriptions are. I could use some help/guidance here on this section.

Reply:
This LGTM. Also had to laugh when I was reading the stimming part because I was fidgeting with my AirPods case while reading.

Reply:
😂 I had several such moments while working on this. Thanks for the follow up!